include/fbgemm/FbgemmBuild.h

```diff
@@ -55,7 +55,7 @@
 #if __clang__ || __GNUC__ >= 4 || __INTEL_COMPILER
 #define ALWAYS_INLINE inline __attribute__((__always_inline__))
 #elif _MSC_VER
-#define ALWAYS_INLINE __forceinline
+#define ALWAYS_INLINE // __forceinline
```

Comment:
Can you tell me why we don't want to force inline in MSVC? Code size? Or some compile errors?

Reply:
This causes infinite compilation time on MSVC at least for me. I am wondering if other people see the same problem.